src/plugins/kibana_usage_collection/README.md

```diff
@@ -3,9 +3,16 @@
 This plugin registers the basic usage collectors from Kibana:
 
 - [Application Usage](./server/collectors/application_usage/README.md)
-- UI Metrics
-- Ops stats
-- Number of Saved Objects per type
-- [User-changed UI Settings](./server/collectors/management/README.md)
-- CSP configuration
 - Core Metrics
+- CSP configuration
```

Comment:
I changed the order of the collectors to match the order with have in the plugin code.